front-end/src/pages/AuthorPage.tsx

```diff
@@ -45,25 +97,21 @@ export default function AuthorPage(props: Props) {
 
   return (
     <Container fluid>
-      <Row>
-        <Col sm={3}>
-          <Card>
-            {/* TODO: maybe add profile pic? uncomment line below */}
+      <Row className="justify-content-md-center">
+        <Col sm={2}>
+          <Card body className="text-center">
+            {/* TODO: maybe add profile pic. uncomment line below */}
             {/* <CardImg top width="100%" src="/assets/318x180.svg" alt="Card image cap" /> */}
             <CardBody>
-              <CardTitle tag="h5">{author.displayName}</CardTitle>
-              <CardSubtitle tag="h6" className="mb-2 text-muted">@{props.loggedInUser?.username}</CardSubtitle>
+              <CardTitle tag="h5">{author?.displayName}</CardTitle>
+              {/* TODO: decide if we want to display/differentiate between username and display name. Results may vary across servers */}
```

Comment:
We can discuss this next meeting, IMO we should just show display names, usernames can be privacy/security sensitive